workshop/exercise-4/README.md

```diff
@@ -1,129 +1,72 @@
 
-### Update domain configuration for Knative
-When a Knative application is deployed, Knative will define a URL for your application. By default, Knative Serving routes use `example.com` as the default domain. The fully qualified domain name for a route by default is `{route}.{namespace}.{default-domain}`, where `{route}` is the name of the application to deploy.
+Because Knative is built on top of Kubernetes, you can use kubectl along with a service definition file to create a new Service definition for your application.
 
-Because we want our application to be accessible at a URL we own, we need to configure Knative to assign new applications to our own domain.
-
-What hostname should we use? Luckily for us, IBM Kubernetes Service gave us an external domain when we created our cluster. We'll first get that URL, tell Knative to assign new applications to that URL, and then forward requests to our Ingress Subdomain to the Knative Istio Gateway.
-
-1. First, let's get the ingress subdomain for our cluster.
-
-	```
-	ibmcloud ks cluster-get <my-cluster-name>
-	```
-
-	Example Output:
-	```
-	Ingress Subdomain:      bmv-knative.us-east.containers.appdomain.cloud   
-	```
-	Ingress is a Kubernetes service that balances network traffic workloads in your cluster by forwarding public or private requests to your apps. This Ingress Subdomain is an externally available and public URL providing access to your cluster. Take note of this Ingress Subdomain.
-
-2. Next, update the default URL for new Knative apps by editing the configuration:
-
-	```
-	kubectl edit cm config-domain --namespace knative-serving
-	```
-
-3. Change all instances of `example.com` to your ingress subdomain, which should look something like: `bmv-knative.us-east.containers.appdomain.cloud`. There should be two instances of `example.com`, one under `data` and one under `annotations`. New Knative applications will now be assigned a route with this host, rather than `example.com`.
-
-### Forward specific requests coming into IKS ingress to the Knative Ingress Gateway
-
-1. When requests come in to our fibonacci application through the ingress subdomain, we want them to be forwarded to the Knative ingress gateway. Update the `forward-ingress.yaml` file with your own ingress subdomain, prepended with `fib-knative.default`. Remember that the fully qualified domain name for a route has the following form: `{route}.{namespace}.{domain}`.
-
-The file should look something like:
-
-```yaml
-  apiVersion: extensions/v1beta1
-  kind: Ingress
-  metadata:
-    name: iks-knative-ingress
-    namespace: istio-system
+1. Create a file named `fib-service.yaml`, with the following contents:
```

Comment:
Any reason we're not just pointing to a file in the repo?  Like we do for other yaml files?

Reply:
we could do that. I was thinking it felt weird to include two service deployment files into an app which could be stand alone.  Especially since we turn around & delete this service anyway.

Reply:
I think it'll be ok - nicer than copy-n-paste.  Even just service1.yaml and service2.yaml is ok.  Just felt odd when we don't ask them to copy-n-paste any place else.